examples/light-switch-app/telink/src/ZclCallbacks.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "AppTask.h"
+
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/ConcreteAttributePath.h>
+#include <lib/support/logging/CHIPLogging.h>
+
+using namespace chip;
+using namespace chip::app::Clusters;
+
+void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath & attributePath, uint8_t mask, uint8_t type,
+                                       uint16_t size, uint8_t * value)
+{
+    ClusterId clusterId     = attributePath.mClusterId;
+    AttributeId attributeId = attributePath.mAttributeId;
+    ChipLogProgress(Zcl, "Cluster callback: " ChipLogFormatMEI, ChipLogValueMEI(clusterId));
+
+    if (clusterId == OnOffSwitchConfiguration::Id)
+    {
+        ChipLogProgress(Zcl, "OnOff Switch Configuration attribute ID: " ChipLogFormatMEI " Type: %u Value: %u, length %u",
+                        ChipLogValueMEI(attributeId), type, *value, size);
```

Comment:
This logs the first byte of the value, right?  Why is that a reasonable thing to do?

Reply:
@bzbarsky-apple 
First byte indicate from which module log output came.
Eg. 
![image](https://user-images.githubusercontent.com/14142428/165263092-425cab8b-a76d-4ff7-9462-b4c68041e491.png)

Reply:
No, I am talking about the fact that `*value` is being logged as `Value: %u`.  That will log the first `uint8_t` of the value.  Why is that OK?  Are all the attributes in this cluster 1-byte integers?

Reply:
You are right about this limitation.
I can improve this to print value by length, but as I see other vendors and some common files has the same situation.
Have I improve this only for Telink Light-Switch app or leave as it is?
![image](https://user-images.githubusercontent.com/14142428/165482949-d4a1267e-827a-4f99-9238-32c800578a25.png)

Reply:
I think the important thing, given this is an example app, is to:

1. Ideally do things that make sense.
2. If doing things that don't make sense because they are placeholders for real processing a real app would do, clearly label them as such in the example.  This way people following the example will know there's a potential pitfall to just blindly copying the example.

So in this case, just clearly documenting that this printing makes no sense for non-single-byte values and a real application would need to do something different here is probably fine.

Reply:
I added comment in both places where this print is used.